egs/swbd/s5c/local/nnet3/run_lstm.sh

```diff
@@ -34,6 +34,7 @@ non_recurrent_projection_dim=256
 chunk_width=20
 chunk_left_context=40
 chunk_right_context=0
+ivector_interval=10
```

Comment:
call this ivector_period. You are using the name ivector_period in other scripts.